pom.xml

```diff
@@ -125,7 +125,16 @@
   <properties>
 
     <!-- project version -->
-    <revision>1.4.0-SNAPSHOT</revision>
+    <revision>1.4.0</revision>
```

Comment:
This shouldn't be necessary since the CI will do it.
Looking at the [`logging-parent` reusable build](https://github.com/apache/logging-parent/blob/27335fbd2af8f377e7d100b51762144258622119/.github/workflows/deploy-release-reusable.yaml#L100), the `git commit` instruction might fail if there is nothing to change.

Reply:
Nope, it is safe. See the `[ "$REVISION" != "$PROJECT_VERSION" ]` check there.

Reply:
So `logging-parent` sets the revision based on the branch name? Will keep that in mind for the future.